include/odp/api/spec/ipsec.h

```diff
@@ -772,6 +772,76 @@ typedef struct odp_ipsec_sa_param_t {
 
 } odp_ipsec_sa_param_t;
 
+/** IPv4 address size */
+#define ODP_IP4ADDR_SIZE 4
+
+/** IPv6 address size */
+#define ODP_IP6ADDR_SIZE 16
+
+/**
+ * Retrieve IPSEC SA information
```

Comment:
This sounds like a description of a function, not a type. Just "SA information" would be better.

Reply:
Agreed. Will update in the next version.